automation/test.sh

```diff
@@ -273,14 +276,14 @@ spec:
 EOF
   # Run only Windows tests
   ginko_params="$ginko_params --ginkgo.focus=Windows"
-elif [[ $TARGET =~ multus.* ]]; then
+elif [[ $TARGET =~ (cnao|multus) ]]; then
```

Comment:
Would it work without .* on the multus side?

Reply:
Yep has tested it.